### DISCUSSION TITLE
MT version for Windows 11 ARM?

### DISCUSSION THREAD

**Discussion by hkrudolph** [mytourbook/mytourbook#1174] [General]
Since my attempts to upgrade macOS version 23.5.1 of MT to 23.8.0 on my M1 Macbook Pro failed, I thought about another alternative: I am also running Windows 11 ARM (as VM in Parallels) on the same M1 Macbook, and have a Java 17 JDK installed in that Win11 VM ... is their a chance that a MT version for Windows on ARM might be available at some point?

**Comment by wolfgang-ch** [comment]
Currently there is no ARM support of Eclipse for Windows but for macOS https://eclipse.dev/eclipse/development/plans/eclipse_project_plan_4_28.xml#target_environments. 

I just created an macOS ARM version for MT 23.8 which you can download from https://www.dropbox.com/scl/fo/3kwz44dzzzkutn4x9pakk/h?rlkey=g62ril9q9p140i3by3ygju5t6&dl=0

I've no idea if this works, just test it.

If this do not work you can still use the Linux version of MT in a VM

**Reply by hkrudolph** [reply]
Since the info.plist of this macOS ARM version did not contain any <string>-vm</string> .... lines to specify JAVA_HOME, I left that .plist as it is ... and just executed the command: `xattr -d com.apple.quarantine /Applications/MyTourbook.app` before a first attempt to start .... MT 23.8 starts, updated the database and seems to work, at first glance!

Many thanks, Wolfgang!

**Comment by wolfgang-ch** [comment]
It's great to hear that it works 😎 

I will provide MT binaries also for macOS ARM starting with the next version of MT. Only a few lines of code https://github.com/mytourbook/mytourbook/commit/923e80a93d18456898c1997abc6fb6ceb4b084b6 were needed to create this additional binary

**Comment by Cnqu6doa2bn** [comment]
Dear Wolfgang,

is there also any chance to get some binaries for aarm64 (Arm64 Linux Arch)?  :)

**Reply by wolfgang-ch** [reply]
These are currently the available platforms:

![image](https://github.com/mytourbook/mytourbook/assets/1283445/f82a896c-2671-4be1-ba14-a328933984bd)

When I read it correctly, then Linux AArch64 is your requested platform?

**Reply by Cnqu6doa2bn** [reply]
Dear Wolfgang,
yes, Linux (64bit for AArch64) would be greatly apreciated! :)

**Reply by wolfgang-ch** [reply]
This platform will be available in the next version https://github.com/mytourbook/mytourbook/commit/6c88c0fa1312ec8068316c27c6fc06392c5eb697

**Comment by Cnqu6doa2bn** [comment]
Dear Wolfgang,

that's very kind of you! Thank you very much :)
Is there any way to send you a donation for your hard work & very useful program?

**Reply by wolfgang-ch** [reply]
> Is there any way to send you a donation for your hard work & very useful program?

When you would like to donate for this project then you can either donate money for a charity organization, which you can select, or you can donate a good algorithm, e.g. https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/tools/tour-chart-smoothing#algorithm which I received 2011

It was an accident that I could add your request (with a few lines of code) so fast, because yesterday morning, I just started the process for the next MT version 24.5 which is available in about 2 weeks.

**Reply by Cnqu6doa2bn** [reply]
Dear Wolfgang,

If I had the mathematical knowledge I would love to help ;)
No worries about the implementation for the Linux-aarm64 version. I can wait and It's very kind from you. Thanks again for your help!
I'll donate to a local charity organisation.
